<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>107</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Фурье преобразование</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QRadioButton" name="rbNoFilter">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Нет фильтра</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbLowFilter">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Низкочастотный фильтр</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbHighFilter">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Высокочастотный фильтр</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Порог для НФ:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Порог для ВФ:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sbLow">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>199.990000000000009</double>
    </property>
    <property name="value">
     <double>80.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sbHigh">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>199.990000000000009</double>
    </property>
    <property name="value">
     <double>20.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>506</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionDo"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionDo"/>
   <addaction name="actionInvDo"/>
   <addaction name="actionSave"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="actionDo">
   <property name="text">
    <string>Выполнить преобразование</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="actionInvDo">
   <property name="text">
    <string>Выполнить обратное преобразование</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
